Sources/XcodeGenKit/ProjectGenerator.swift

```diff
@@ -104,8 +105,8 @@ public class ProjectGenerator {
 
                     let debugConfig = spec.configs.first { $0.type == .debug }!
                     let releaseConfig = spec.configs.first { $0.type == .release }!
-
-                    let specScheme = Scheme(name: schemeName, targets: [Scheme.BuildTarget(target: target.name)], debugConfig: debugConfig.name, releaseConfig: releaseConfig.name)
+                    
```

Comment:
Not a big deal as I run `make format_code` before a release, but I guess our whitespace settings differ

Reply:
I hadn't seen it, change undone. 👍